### PR TITLE
Source spec from {{Specifications}} for missed api pages

### PR DESCRIPTION
**Pull request by teoli2003**
Use {{Specifications}} on missed API Pages (because no bcd key, H3 instead of H2, wrong headers).

Part of #1146.

### CODE REVIEW

File: files/en-us/web/api/gamepad/index.html [hunk 1]
```diff
@@ -25,30 +25,23 @@ <h2 id="Properties">Properties</h2>
...
  <dd>An integer that is auto-incremented to be unique for each device currently connected to the system.</dd>
  <dt>{{domxref("Gamepad.mapping")}} {{readonlyInline}}</dt>
  <dd>A string indicating whether the browser has remapped the controls on the device to a known layout.</dd>
+ <dt>{{domxref("Gamepad.pose")}} {{readonlyInline}}{{ExperimentalInline}}</dt>
+ <dd>A {{domxref("GamepadPose")}} object representing the pose information associated with a WebVR controller (e.g. its position and orientation in 3D space).</dd>
  <dt>{{domxref("Gamepad.timestamp")}} {{readonlyInline}}</dt>
  <dd>A {{domxref("DOMHighResTimeStamp")}} representing the last time the data for this gamepad was updated.</dd>
 </dl>
 
-<h3 id="Experimental_extensions_to_Gamepad">Experimental extensions to Gamepad</h3>
```
Comment (Elchi3) [inline files/en-us/web/api/gamepad/index.html:39]: Does this change belong to this PR?

**Review by Elchi3 — APPROVED** [review]
Looks good just wondering if the Gamepad changes should be here

File: files/en-us/web/api/gamepad/index.html (hunk 1 above)
Comment (teoli2003) [reply files/en-us/web/api/gamepad/index.html:39]: I was hesitating in creating a new PR: my goal was to remove the {{Spec2}}, {{SpecMacro}}, so I went forward to incorporate them with all the properties, like we do almost everywhere. The individual pages have clear indication about them being experimental.

I can remove this interface from this PR, if you prefer.

Comment (Elchi3) [reply files/en-us/web/api/gamepad/index.html:39]: I agree with the changes, but yes, let's have it in a separate PR.

Comment (teoli2003) [reply files/en-us/web/api/gamepad/index.html]: Done; I will create a PR with several {{Spec2}} {{SpecName}} removal outside bcd tables.

**Review by Elchi3 — APPROVED** [review]

**Pull request merged**